src/shared/components/Modal/Modal.tsx

```diff
@@ -108,6 +109,11 @@ export const Modal = ({
           )}
         </StyledDialogTitle>
         {children}
+        {footer && (
+          <StyledFlexSpaceBetween sx={{ gap: 1.6, px: 3.2, pt: 3.6, pb: 2.4 }}>
```

Comment:
Does it make sense to also apply `footerStyles` here, given its name? Or perhaps rename that prop to `actionsStyles`? Apparently it's no longer even used in the codebase, probably a relic after some restyling was done. Maybe we could delete it?

Reply:
It looks like there's actually one instance of `footerStyles` being used, here in `RecordAudio.tsx`:

https://github.com/ChildMindInstitute/mindlogger-admin/blob/fcc80a7eaa39605c4452f60a51d65a632d3f8ed0/src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/AudioPlayer/RecordAudio/RecordAudio.tsx#L76-L78

I kept going back-and-forth on this — My original thinking was that this wrapper element around the `footer` content should just be a sensible default that allowed for the what seemed like the common button configurations design was moving towards (per their demo a few weeks back) — single buttons are centered, multiple buttons are arranged with the `space-between`. 

For any more nuanced cases and discrete groups (which I'm sure there's a lot of — even the popup I added here was a bit a typical with multiple centered buttons), devs can just provide their own wrappers to create whichever layouts they please.

That said, `footerStyles` can sure be handy, if all you need is a different `justifyContent` property.

I'm loathe to add even more props to this component, though. If we're opening up that wrapper to configuration, I'd be more inclined to just have `footerProps` or similar, and spread that to this element, instead of inevitably adding more props for configuring each individual attribute as the need arises.

But that also seems like overkill to me, especially since I wouldn't even really need this in my PR here.

In the end I opted to just pass `footerStyles`. 😅

Reply:
Works for me! Thanks for sharing your thought process.

FWIW I think all options for button flexibility could be handled by a single array of ButtonProps objects, which is what we did on my last project for a very similar situation, and it worked great. It was very flexible, allowing them to dictate variants, disabled state, click events, and layout could be dictated by just providing left & right `auto` margins as needed, which allows you to centre them as well as lay them out in a space-between fashion. Some future refactor, perhaps. 🙂